File: test/test-assign/r-correct/student/tests/q1.R
test = list(
  name = "q1",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 1,
      code = {
        
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 1,
      code = {
        testthat::expect_true(x < 100)
      }
    )


  )
)

File: test/test_run/files/autograder/source/tests/q1.py
OK_FORMAT = True

test = {
	"name": "q1",
	"points": 0,
	"hidden": False,
	"suites": [ 
		{
			"cases": [ 
				{
					"code": r"""
					>>> square(3)
					9
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> square(2.5)
					6.25
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> square(6)
					36
					""",
					"hidden": True,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> square(1.5)
					2.25
					""",
					"hidden": True,
					"locked": False,
				}, 
			],
			"scored": False,
			"setup": "",
			"teardown": "",
			"type": "doctest"
		}
	]
}